splink/comparison_level_library.py

```diff
@@ -233,3 +261,61 @@ def create_label_for_charts(self) -> str:
         return (
             f"Jaro-Winkler distance of '{self.col_name} >= {self.distance_threshold}'"
         )
+
+
+class JaroLevel(ComparisonLevelCreator):
+    def __init__(self, col_name: str, distance_threshold: Union[int, float]):
+        """A comparison level using a Jaro distance function
+
+        e.g. `jaro(val_l, val_r) >= distance_threshold`
+
+        Args:
+            col_name (str): Input column name
+            distance_threshold (Union[int, float]): The threshold to use to assess
+                similarity
+        """
+
+        super().__init__(col_name)
+        self.distance_threshold = validate_distance_threshold(
+            lower_bound=0,
+            upper_bound=1,
+            distance_threshold=distance_threshold,
+            level_name=self.__class__.__name__,
+        )
+
+    def create_sql(self, sql_dialect: SplinkDialect) -> str:
+        col_l, col_r = self.input_column(sql_dialect).names_l_r()
```

Comment:
This won't work on this branch, but it will downstream on Splink4_dev

```suggestion
        col_l, col_r = self.input_column(sql_dialect).names_l_r
```